docs/Homebrew-brew-Maintainer-Guide.md

```diff
@@ -0,0 +1,105 @@
+# Homebrew/brew Maintainer Guide
+
+This document describes a few components of the `Homebrew/brew` repository that are useful for maintainers to
+be aware of, but don't necessarily need to appear in documentation for most users and contributors.
+
+## Reviewing PRs
+
+Using `gh pr checkout NUMBER` is a super easy way to check out a PR branch using the GitHub CLI.
```

Comment:
Might be helpful to mention that this is aliased to `gh co` by default. Or mention this very useful alias:
```
gh alias set co --shell 'id="$(gh pr list -L100 | fzf | cut -f1)"; [ -n "$id" ] && gh pr checkout "$id"'
```
from https://github.blog/2021-03-11-scripting-with-github-cli/

Reply:
I agree this is super useful (I've used it regularly since you mentioned it to me) but I think I'd rather stick to the non-aliases for the guide.